packages/@webex/plugin-meetings/src/meetings/util.ts

```diff
@@ -90,7 +90,8 @@ MeetingsUtil.getMediaServer = (sdp) => {
       .find((line) => line.startsWith('o='))
       .split(' ')
       .shift()
-      .replace('o=', '');
+      .replace('o=', '')
+      .toLowerCase();
```

Comment:
nice. Hopefully we added a test for the casing

Reply:
added the test now